src/explorer.ts

```diff
@@ -20,13 +20,28 @@ export function createKubernetesResource(kind: kuberesources.ResourceKind, id: s
 }
 
 function getIconForPodStatus(status: string): vscode.Uri {
-    if (status === "running" || status === "completed") {
+    if (status === "Running" || status === "Completed") {
         return vscode.Uri.file(path.join(__dirname, "../../images/runningPod.svg"));
     } else {
         return vscode.Uri.file(path.join(__dirname, "../../images/errorPod.svg"));
     }
 }
 
+function parsePodLine(podInfo: string[]): any[] {
```

Comment:
Looking at this function, I don't think it has anything to do with pods specifically - you have designed it to be completely generic.  So I think we should consider moving it into `kubectl` or `kubectlUtils` as a generic "get me a parsed representation of a tabular resource list" function.  In fact it would even work with Helm...

Reply:
Cool, will move it to `kubectlUtils` if that is ok.